### ISSUE TITLE
Tribal Nations search widget vs. screenreaders

### ISSUE THREAD

**Issue by lfrancis** [usnationalarchives/nara-bia-photos#140]
https://nara-bia-photos.herokuapp.com/tribal-nations

The search widget doesn't announce its matches and so is not accessible to screenreader users:
![image](https://user-images.githubusercontent.com/6942589/96000881-6100cd80-0e05-11eb-83ca-7579e3d4d9b9.png)
